core/src/test/scala/unit/kafka/server/GroupCoordinatorBaseRequestTest.scala

```diff
@@ -660,6 +660,24 @@ class GroupCoordinatorBaseRequestTest(cluster: ClusterInstance) {
     )
   }
 
+  protected def classicLeaveGroup(
```

Comment:
what's the difference between this and leaveGroupWithOldProtocol except for the expected response?

Reply:
can't we call `classicLeaveGroup` inside `leaveGroupWithOldProtocol` to reuse the code?

Reply:
yes, we can. let me update that too.